### PR TITLE
Fix ssd1306 font in ssd1306_i2c example

### LINKED ISSUE #403: SSD1306 font is upside down in I2C SSD1306 example
See comment on line 286, https://github.com/raspberrypi/pico-examples/blob/master/i2c/ssd1306_i2c/ssd1306_i2c.c.

### PR DESCRIPTION
**Pull request by kuleshmaria**
Fixes #403.

### CODE REVIEW

**Comment by lurch** [comment]
ping @JamesH65

File: i2c/ssd1306_i2c/ssd1306_font.h [hunk 1]
```diff
@@ -7,42 +7,44 @@
 // Vertical bitmaps, A-Z, 0-9. Each is 8 pixels high and wide
 // Theses are defined vertically to make them quick to copy to FB
 
+// Update: Reverse font
```
Comment (lurch) [inline i2c/ssd1306_i2c/ssd1306_font.h:10]: This comment probably doesn't make sense when viewed outside the context of this PR? (i.e. it should probably be deleted?)

Comment (JamesH65) [reply i2c/ssd1306_i2c/ssd1306_font.h:10]: Yes, it should be deleted in the context of this PR.

**Comment by kuleshmaria** [comment]
Fixed it!

**Review by JamesH65 — APPROVED** [review]
LGTM

**Pull request merged**